### PR TITLE
297 style resource pages

### PR DESCRIPTION
**Pull request by Nikomus**
Trello card: https://trello.com/c/h4fXlidL/297-add-basic-styles-to-help-self-resource-pages

## Description

- Styled resource pages

image

@neontribe/sea-map

### CODE REVIEW

File: src/Page/Definition.elm [hunk 1]
```diff
@@ -329,8 +329,8 @@ expanderDefinitionStyle =
     ]
 
 
-quoteStyle : List Style
-quoteStyle =
+quoteStyles : List Style
```
Comment (Nikomus) [inline src/Page/Definition.elm:325]: I'm trying to be consistent with naming `xStyles` for List Style as a hint it's a list of them, not one.

File: src/Page/HelpSelfSingle.elm [hunk 2]
```diff
@@ -71,21 +73,28 @@ view slug model =
 
 renderResourceList : Model -> List CategoryResource -> Html Msg
 renderResourceList model resources =
-    ul []
+    ul [ css [] ]
```
Comment (katjam) [inline src/Page/HelpSelfSingle.elm]: Do you want to keep this?

File: src/Page/HelpSelfSingle.elm [hunk 3]
```diff
@@ -95,13 +104,14 @@ renderResourceDetails : CategoryResource -> List (Html msg)
...
-           , a [ href (t resource.linkHref) ] [ text (t resource.linkName) ]
+           , verticalSpacing
+           , a [ href (t resource.linkHref), css [ resourceLinkStyle ] ] [ text (t resource.linkName) ]
            ]
 
 
 renderQuotes : List Key -> List (Html msg)
 renderQuotes quoteKeys =
-    List.map (\quoteKey -> p [] [ text (t quoteKey) ]) quoteKeys
+    List.map (\quoteKey -> p [ css quoteStyles ] [ text (t quoteKey) ]) quoteKeys
```
Comment (katjam) [inline src/Page/HelpSelfSingle.elm]: blockquote or p?

**Review by katjam — COMMENTED** [review]
Couple of minor questions.

File: src/Page/HelpSelfSingle.elm [hunk 4]
```diff
@@ -71,37 +73,46 @@ view slug model =
...
         )
 
 
 renderResourceDetails : CategoryResource -> List (Html msg)
 renderResourceDetails resource =
-    renderQuotes resource.quotes
-        ++ [ p [] [ text (t resource.summary) ]
-           , a [ href (t resource.linkHref) ] [ text (t resource.linkName) ]
-           ]
+    [ blockquote [] ([] ++ renderQuotes resource.quotes)
```
Comment (katjam) [inline src/Page/HelpSelfSingle.elm]: I think the blockquote needs to be around each one since they are not same person.

**Review by katjam — CHANGES_REQUESTED** [review]

**Review by katjam — APPROVED** [review]
Blockquote styling still little confusing - but happy to go with this! :rabbit:

**Pull request merged**